image/templates/helm/stackrox-secured-cluster/values-public.yaml.example

```diff
@@ -299,12 +299,10 @@
 ## Collector specific configuration.
 #collector:
 #
-#  # If enabled, the value provided in `collectionMethod` is applied to the collector configuration.
-#  # Otherwise, the default collection method is used.
+#  # Deprecated. This value has no effect.
```

Comment:
I would suggest to remove it here. This file is supposed to serve as an configuration example for new users. I think it would not provide any value to the user if it includes deprecated config options without any effect.